htsget-config/README.md

```diff
@@ -431,10 +431,53 @@ following JSON structure, indicating whether the request is allowed, and any reg
+ids like the [regex resolvers](#regex-based-location). A full JSON schema defining this format is available under [auth.json][auth-json].
+An [example][auth-example] configuration file is available in the examples directory.
+
+With this authorization logic, the server will respond with a `403 Forbidden` error if any of the requested reference
+names are not allowed according to the restrictions. For example, if using the above JSON restrictions, a user
+requesting `?referenceName=chr1&format=BAM&start=500&end=1500` will receive a `403` error, even though part of the range
+is satisfiable (i.e. according to the restrictions, from `start=500` to `end=1000`). In order to address this issue, the
+following flag can be enabled under the `auth` table:
+
+| Option            | Description                                                                                      | Type    | Default |
+|-------------------|--------------------------------------------------------------------------------------------------|---------|---------|
+| `suppress_errors` | Return any available regions according to restrictions, even if the full request is not allowed. | Boolean | `false` |
+| `add_hint`        | Add a hint to the ticket response that indicates which regions the client is allowed to view.    | Boolean | `true`  |
+
+To enable this option, htsget-rs needs to be compiled with `--features experimental` as suppressed errors may lead to
+less regions being returned than requested, which may not follow the htsget protocol. When this option is used, the
+above example with the request: `?referenceName=chr1&format=BAM&start=500&end=1500`, would return reads between `start=500`
+and `end=1000` for the reference name. Additionally, there will be another field present in the JSON response which hints
+to clients that the full range was not satisfied because of authorization restrictions:
+
+```json
+{
+  "htsget": {
+    "format": "BAM",
+    "urls": [<list of urls...>],
+    "allowed": [
+      {
+        "name": "chr1",
+        "format": "BAM",
+        "start": 100,
+        "end": 1000
+      }
+    ] 
+  }
+}            
+```
+
+The `allowed` field echos the rule defined in the restrictions, and allows clients to plan for a partially returned
+response. This field can be removed by setting `add_hint = false` in the `auth` table. 
+
+Note that when using `suppress_errors = true`, when a client is not authorization to view any of the requested regions,
+a non-error response is still returned with an empty reads/variants set. For example, if the client was not authorized
+to view `chr1` at all, the response would return urls that correspond to a valid file (with the file header and EOF block),
+but no actual reads data. Authorization error responses are still returned if the JWT is invalid or the client is not
+allowed to view the requested `<id>` path.
```

Comment:
Are there any restrictions of the requests that will cause an error? negative locations, chromosome names? If not, maybe a sentence indicating that not just the `chr1` will not cause an error - but that filter parameters are _not_ checked for validity.